pkgs/top-level/all-packages.nix

```diff
@@ -575,6 +575,16 @@ with pkgs;
 
   mod = callPackage ../development/tools/mod { };
 
+  mysql-shell = callPackage ../development/tools/mysql-shell {
+    inherit (darwin) cctools developer_cmds DarwinTools;
+    inherit (darwin.apple_sdk.frameworks) CoreServices;
+    boost = boost173; # Configure checks for specific version.
+    protobuf = protobuf3_11;
+    icu =  icu67;
+    v8 = v8_8_x;
+    python = python39;
```

Comment:
```suggestion
```

Reply:
It needs to be Python 3. How about `python = python3` ?